lib/testUtils/createRuleTester.js

```diff
@@ -160,7 +160,7 @@ function processGroup(rule, schema, equalityCheck) {
     processor.use(assignDisabledRanges)
 
     if (schema.preceedingPlugins) {
-      schema.preceedingPlugins.forEach(processor.use)
+      schema.preceedingPlugins.forEach(processor.use.bind(processor))
```

Comment:
can also make this a `schema.preceedingPlugins.forEach(plugin => processor.use(plugin))` if that's favorable